Untitled.ipynb

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This should be removed

Reply:
Ups, I thought notebooks were included on the gitignore. Already solved